### PR TITLE
Add duplicate detector CLI

### PR DESCRIPTION
**Pull request by rdar-lab**

### CODE REVIEW

File: src/build_executable.py [hunk 1]
```diff
@@ -310,14 +329,21 @@ def main():
...
     cli_success = build_with_pyinstaller(spec_file_cli)
 
     if not cli_success:
         logger.error("[FAILED] CLI build failed!")
         sys.exit(1)
 
+    logger.info("Building Duplicate Detector executable...")
+    dup_dd_success = build_with_pyinstaller(spec_file_duplicate_detector)
+
+    if not dup_dd_success:
```
[the remark this answers is not in the document]
Comment (rdar-lab) [reply src/build_executable.py]: Not critical

**Comment by rdar-lab** [comment]
@copilot open a new pull request to apply changes based on the comments in [this thread](https://github.com/rdar-lab/convert_videos/pull/47#pullrequestreview-3673679694)

Also please extend the test live docker to create and test the docker based on the duplicate detection (create a temp dir with twice the same video - you can use the same video in the fixtures directory, and run the docker and validate the result)
